.github/workflows/stale.yaml

```diff
@@ -0,0 +1,56 @@
+# This workflow alerts and then closes the stale issues/PRs after specific time
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+
+name: 'Close stale issues and PR'
+on:
+  schedule:
+    - cron: "30 1 * * *"
+    
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v7
+        with:
+          # comment on issue if not active for more then 14 days.
+          stale-issue-message: 'This issue has been automatically marked as stale because it has no recent activity. It will be closed if no further activity occurs. Thank you.'
+        
+          # comment on PR if not active for more then 14 days.
+          stale-pr-message: 'This PR has been automatically marked as stale because it has no recent activity. It will be closed if no further activity occurs. Thank you.'
+        
+          #comment on issue if stale for more then 14 days. 
+          close-issue-message: "Closing as stale. Please reopen if you'd like to work on this further"
+        
+          #comment on PR if stale for more then 14 days. 
+          close-pr-message: "Closing as stale. Please reopen if you'd like to work on this further"
+        
+          # Number of days of inactivity before an Issue Request becomes stale
+          days-before-issue-stale: 14
+         
+          # Number of days of inactivity before a stale Issue is closed
+          days-before-issue-close: 14
+          
+          # reason for closed the issue default value is not_planned 
+          close-issue-reason: completed
```

Comment:
Why we want to change it to `completed` ?

Reply:
Else, it would close the issue like this https://github.com/tensorflow/tensorflow/issues/59699

Reply:
<img width="907" alt="image" src="https://user-images.githubusercontent.com/124146945/219571207-12412810-1231-4cb9-91b0-fba1b65fdfd4.png">

if we will not select it as completed then it will show as not_planned that is the default value of close-issue reason .